### ISSUE TITLE
Activation code modal: don't check if email is used

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#626]
As a user, if I fill my email address, but for some reason don't complete with filling in my activation code the first time, the next time I try I get the message:

<img width="429" alt="screenshot 2018-12-31 at 12 11 06" src="https://user-images.githubusercontent.com/30194799/50559241-2fcd1100-0cf5-11e9-8949-bcdc0fceaa7b.png">

But now I have an account with no activated vouchers. What the user needs to do now to activate the voucher is:
1. Log in
2. Click start met activeren
3. Activate the voucher

It might not be entirely clear to the user that these are the required steps

## Proposal
If the user already has an account, but no vouchers (or even if they do have vouchers) allow them to "log in" trough the same flow as new users.
1. Click start met activeren
2. Open link in email
3. Activate the voucher

This flow is clearer as you are "taken by the hand" trough the 3 steps, in the situation above the user will have to figure this out for themselves.

So in short: don't check if the email address has already been used.

**Comment by maxvisser** [comment]
Good suggestion
